### PR TITLE
Fix function stack L0 test

### PR DESCRIPTION
**Pull request by debermudez**
Fixing the function stack test for python 3.8

### CODE REVIEW

**Review by tgerdesnv — CHANGES_REQUESTED** [review]
Please change compare_funcstack() instead of each individual case. That function already attempts to rip off the start of the string.  Change it to rip off to _callTestMethod instead of run.

**Comment by debermudez** [comment]
Updated

File: qa/L0_function_stack/test_pyprof_func_stack.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def remove_test_class_hierarchy(x):
             # Find the LAST instance of run in the split
             #
             for i, n in enumerate(fn_split):
-                if (n == "TestPyProfFuncStack::run"):
+                if (n == "TestPyProfFuncStack::_callTestMethod"):
```
Comment (tgerdesnv) [inline qa/L0_function_stack/test_pyprof_func_stack.py:52]: Comment on line 43 needs to be updated or generalized

**Review by tgerdesnv — CHANGES_REQUESTED** [review]

File: qa/L0_function_stack/test_pyprof_func_stack.py (hunk 1 above)
Comment (debermudez) [reply qa/L0_function_stack/test_pyprof_func_stack.py:52]: done

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**